### ISSUE TITLE
Event form - list of changes to perform

### ISSUE THREAD

**Issue by phpgeekfr** [blockframes/blockframes#7134]
- [x] Screening edit page - add an Asterisk to the Title selection as it is mandatory #7112 
- [x] Event form - Add a chip to warn users there is no screener #6978 

- [x] Page improvements #7178
 ![image](https://user-images.githubusercontent.com/1194344/141968863-3264d7b7-6f35-4dfa-9c18-2ec6cf9b0d50.png)

- [x]  https://github.com/blockframes/blockframes/issues/7061#issuecomment-969037617
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/21e2bdcd-eb21-4372-9b59-84675aa543a8)
- [x] On event edit page, invitations should be sent only after saving the event form 
- [ ] Not related to events only but could improve the form : Issue on animations #6778
- [ ] New event form auto-scroll on top #6759
- [x] Find a better way to check if eventForm is dirty #6780
- [x] Popin when leaving event form without saving : Blur effect missing on the background
- [x] Add an "i" tooltip icon otherwise the tooltip text is way too low to understand which line it refers to. 
![image](https://user-images.githubusercontent.com/55785823/141771074-f2d1eeb3-25c1-483e-a79c-22c2089ed43f.png)
![image](https://user-images.githubusercontent.com/55785823/141771092-da14a7c1-d3d4-40c4-9ec0-1bd74d7d9819.png)

- [x] On protected events, when I invite someone with an account, the status appears as pending but when I invite someone without, it appears as accepted, which is a bit confusing:
![image](https://user-images.githubusercontent.com/55785823/142193724-31f08af7-a5bd-4f10-808d-796a131f2b43.png)

_Originally posted by @maltmayer in https://github.com/blockframes/blockframes/issues/7122#issuecomment-970539764_

- [x] Small improvements for Event Form UI #7171

**Comment by cams-goussale** [comment]
> Screening edit page - add an Asterisk to the Title selection as it is mandatory Screening edit page - add an Asterisk to the Title selection as it is mandatory #7112

2 points of this issue are contradictory: we want organizers to be able to create a screening without a video and just warn them the video is missing. The Title seleection field shouldn't be mandatory.

I'll mark as closed the asterisk issue #7112. Intead we need to reopen this point in new issue #7178:

>This page should be improved visually. Have an illustration or at least a CTA to contact us or the sales company
image
![image](https://user-images.githubusercontent.com/93206363/142406305-bbdc9145-8a8e-48b2-80ff-3e0558516133.png)

**Comment by phpgeekfr** [comment]
Referencing #7138 so we can take it into account and think about it when handling this issue
